client/src/components/CreateAuction.jsx

```diff
@@ -84,50 +88,163 @@ class CreateAuction extends React.Component {
       })
   }
 
+  handleImageInput(e) {
+    this.setState({
+      image_url: e.target.value
+    })
+  }
+
+  handleImageCreate() {
+    var formData = new FormData();
+    formData.append('image_file', document.getElementById('imageToSend').files[0]);
+    fetch('/images', {
+        method: 'post',
+        body: formData
+    })
+    .then((data) => {
+      console.log('posting image to Cloudinary SUCCESS! data: ', data);
```

Comment:
No sorries. Haha, I got some of mine in the PR as well. Just a reminder for all of us.